elastic/kibana
# [HTTP/OAS] Commit OAS snapshot

Close https://github.com/elastic/kibana/issues/181992

## Summary

First iteration of a CLI to capture an OAS snapshot.

## How to test

Run `node ./scripts/capture_oas_snapshot.js --update --include-path /api/status` and see result in `oas_docs/bundle.json`.

If you have the [bump CLI](https://www.npmjs.com/package/bump-cli) installed you can preview the hosted output with `bump preview ./oas_docs/bundle.json`

## Notes
* Added ability to filter by `version`, `access` (public/internal) and excluding paths explicitly to the OAS generation lib
* Follows the same general pattern as our other "capture" CLIs like `packages/kbn-check-mappings-update-cli`
* Result includes only `/api/status` for now, waiting for other paths to add missing parts
